Lib/fontTools/feaLib/testdata/spec9e.fea

```diff
@@ -1,4 +1,6 @@
 table name {
-  nameid 9 "Joachim M\00fcller-Lanc\00e9";    # Windows (Unicode)
-  nameid 9 1 "Joachim M\9fller-Lanc\8e";      # Macintosh (Mac Roman)
+#test-fea2fea: nameid 9 "Joachim Müller-Lancé";
```

Comment:
Yes. Feel free to add the processing (I would ignore the Mac encoding stuff, and just go with Unicode). Note that \ needs to be encoded as \005C. If you want me to do this, I'm happy to do this early next year. If so, raise a bug.